resequence service credential loading

d047491 opened the pull request.

jkbschmid commented:
> Error: unnecessary leading newline (whitespace)

⚠️ Could you please fix lint issues and add tests?⚠️ 

Just kidding - still had notifications turned on for this repo, but I'll leave you alone now 😉

d047491 commented:
> > Error: unnecessary leading newline (whitespace)
> 
> ⚠️ Could you please fix lint issues and add tests?⚠️
> 
> Just kidding - still had notifications turned on for this repo, but I'll leave you alone now 😉

:-D Hahaha . Nice to hear from you. and thanks for the review. I added tests and fixed the linting <3
